### ISSUE TITLE
pip install error

### ISSUE THREAD

**Issue by vintersnow** [shap/shap#55]
I'm not sure why, but I couldn't install from pip.
I try below command

```
$ pip install --user shap
shap/_cext.cc:5:10: faltal error: tree_shap.h: No such file or directory
     #include "tree_shap.h"
```

I was able to install from setup.py

```
$ python setup.py install --user
```

**Comment by Benno26** [comment]
Same regarding upgrading using 'pip install shap --upgrade':

```
 shap/_cext.cc:5:10: fatal error: 'tree_shap.h' file not found
    #include "tree_shap.h"
             ^~~~~~~~~~~~~
    1 error generated.
    error: command 'gcc' failed with exit status 1
```

Worked fine when installing from directly from git repo though:

`pip install git+git://github.com/slundberg/shap.git`

**Comment by Benno26** [comment]
Worked fine when installing from directly from git repo though:

`pip install git+git://github.com/slundberg/shap.git`

**Comment by slundberg** [comment]
Thanks for letting me know. Travis does a github checkout, so that's why it didn't catch it. Fixed now in 0.13.6
